src/gui/symbology-ng/qgssymbolv2selectordialog.cpp

```diff
@@ -110,31 +110,31 @@ class SymbolLayerItem : public QStandardItem
     QVariant data( int role ) const override
     {
       if ( role == Qt::DisplayRole || role == Qt::EditRole )
+  {
```

Comment:
remember to run scripts/prepare_commit.sh to fix the spacing before committing, that should avoid unnecessary changes like these